homeassistant/components/decora_wifi/light.py

```diff
@@ -96,6 +97,13 @@ class DecoraWifiLight(LightEntity):
     def __init__(self, switch):
         """Initialize the switch."""
         self._switch = switch
+        self.uid = None
+
+    @property
+    def unique_id(self):
+        """Return a unique per-device ID based on MAC address and serial number."""
+        self.uid = f"{format_mac(self._switch.mac)}-{self._switch.serial}"
```

Comment:
Maybe the serial number is enough as unique id? Do we need to use both mac address and serial number?